src/main/java/org/codehaus/mojo/taglist/ReportGenerator.java

```diff
@@ -122,6 +122,8 @@ private void doSummarySection(Collection<TagReport> tagReports) {
         sink.paragraph_();
 
         sink.table();
+        sink.tableRows(new int[] {}, false);
```

Comment:
You can make it more compact: https://github.com/apache/maven-doxia/blob/45aaa052ec62528a34c7bd2900764248e60fc331/doxia-core/src/main/java/org/apache/maven/doxia/sink/impl/AbstractSink.java#L289